.travis.yml

```diff
@@ -26,9 +26,11 @@ env:
     global:
         - NUMPY_VERSION=1.10
         - ASTROPY_VERSION=1.1
+        - LIBGFORTRAN_VERSION=1.0
         - CONDA_DEPENDENCIES='Cython click scipy h5py matplotlib pyyaml scikit-image scikit-learn pandas pyregion naima photutils wcsaxes sherpa'
+        - CONDA_DEPENDENCIES_LINUX='$CONDA_DEPENDENCIES libgfortran=$LIBGFORTRAN_VERSION'
```

Comment:
@cdeil - at the moment I think this is the best logic. However this won't work, as the shell substitutions are a bit weird and non working from the yaml file. So you have to list all the dependencies, without reusing an env variable.

So I would suggest to have one ``CONDA_DEPENDENCIES`` that is for linux (as that's the widest used one), and then another one for ``CONDA_DEPENDENCIES_OSX`` which is basically a copy of the former without the libgfortran.